### ISSUE TITLE
How can install pyG in a offline environment ?

### ISSUE THREAD

**Issue by LEE-JYOUNG** [rusty1s/pytorch_geometric#2875]
From now on, we recommend using our discussion forum (https://github.com/rusty1s/pytorch_geometric/discussions) for general questions.

## ❓ Questions & Help

I have to install pyG in a offline GPU server, i have downloaded cluster/sparse/scatter/spline-cov whl files and installed with pip commands, but how can install pyG ? i cannot find any guidence

**Comment by rusty1s** [comment]
You can download the latest PyG version from [here](https://pypi.org/project/torch-geometric/).

**Comment by jlevy44** [comment]
The tar.gz file did not work for one of my collaborators. Would it be possible to generate some wheels for windows?

**Comment by jlevy44** [comment]
@rusty1s

**Comment by rusty1s** [comment]
I don't think that dependencies are bundled when providing wheels, and `pip` will still install PyG dependencies from the internet. I think one workaround is to follow this [guide](https://idomagor.medium.com/short-and-hopefully-simple-guide-to-install-python-libraries-to-a-offline-station-computer-ad8fd2757921). Let me know if it helps.
